src/main/java/cuchaz/enigma/translation/mapping/serde/EnigmaMappingsReader.java

```diff
@@ -149,7 +149,7 @@ private int countIndentation(String line) {
 				return parseMethod(parentEntry, tokens);
 			case "arg":
 				return parseArgument(parentEntry, tokens);
-			case "javadoc":
+			case EnigmaFormat.COMMENT:
```

Comment:
note: this has been lowercased and this will never match as a result.

Reply:
COMMENT is in lower case

Reply:
See commit “oops”

Reply:
Oh, there’s a bunch of indents i need to fix in that commit

Reply:
actually consider changing everything in the switch to use upper case. that is how enigma writes the identifiers

Reply:
sure